src/opnsense/scripts/captiveportal/cp-background-process.py

```diff
@@ -261,13 +231,10 @@ def main():
 
             # process sessions per zone
             for zoneid in bgprocess.list_zone_ids():
-                bgprocess.sync_zone(zoneid)
+                bgprocess.sync_zone(zoneid, IPFW.list_accounting_info())
```

Comment:
best move `list_accounting_info()` up so we can reuse it in the loop and `update_accounting_info()` below.